src/core/src/op/group_normalization.cpp

```diff
@@ -10,6 +10,8 @@
 #include "openvino/core/validation_util.hpp"
 
 namespace ov {
+op::v12::GroupNormalization::GroupNormalization() : m_num_groups{0}, m_epsilon{0} {}
```

Comment:
Consider setting default values to members in class definition -> One file changed and ctor remains `default`.

Reply:
I don't find it as an argument.